File: official/docs/node/v6/pagination/get-next-page.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient(process.env.EASYPOST_API_KEY);

(async () => {
  // Get first page of results
  const shipments = await client.Shipment.all({
    page_size: 5,
  });

  // Provide the previous results page to move onto the next page
  const nextPage = await client.Shipment.getNextPage(shipments);

  console.log(nextPage);
})();

File: official/docs/csharp/v6/pagination/get-next-page.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            // Get first page of results
            EasyPost.Parameters.Shipment.All parameters = new()
            {
                PageSize = 5
            };

            EasyPost.Models.API.ShipmentCollection shipmentCollection = await client.Shipment.All(parameters);

            // Provide the previous results page to move onto the next page
            EasyPost.Models.API.ShipmentCollection nextPage = await client.Shipment.GetNextPage(shipmentCollection);

            Console.WriteLine(JsonConvert.SerializeObject(nextPage, Formatting.Indented));
        }
    }
}
